tests/data/modules/test_674/plugins/__init__.py

```diff
@@ -24,3 +24,18 @@ def test_not_nullable_list(param: "int[]") -> "bool":
 @plugin
 def test_returns_none() -> "string?":
     return None
+
+
+@plugin
+def test_float_to_int(val1: "float") -> "int":
+    return int(val1)
+
+
+@plugin
+def test_int_to_float(val1: "int") -> "float":
+    return float(val1)
+
+
+@plugin
+def test_error_float() -> "float":
+    return "hello"
```

Comment:
Here too I would rather opt for the `int` than `str`. No need to try both I think, just the least trivial one should suffice.